.editorconfig

```diff
@@ -1,2 +1,4 @@
 [*.{gradle,java}]
 indent_style = tab
+ij_continuation_indent_size = 8
+ij_java_imports_layout = $*,|,java.**,|,*,|,net.fabricmc.**
```

Comment:
Nice! I didnt know this was possible. 👍

Reply:
It's a pretty nice IDEA-specific editorconfig feature. (Could be added to Fabric API as well since its import layout quite far from IDEA's default)